kod/object/active/holder/nomoveon/battler/monster.kod

```diff
@@ -5827,17 +5819,9 @@ messages:
       return;
    }
 
-   % True if the monster is ready to say a welcoming message to players
-   %  who have entered their room.
    ReadyForWelcome()
```

Comment:
a) This function doesn't no longer returns TRUE or FALSE, which looks like a bug.
b) Is this function still needed?  It looks like you now only call it on Towns.